doc/source/cluster/kubernetes/user-guides/ray-k8s-setup.md

```diff
@@ -0,0 +1,81 @@
+(ray-kubernetes-setup)=
+
+# Set up a Ray + Kubernetes cluster
+
+This document contains recommendations for setting up a Ray + Kubernetes cluster for your organization.
+
+When you set up Ray on Kubernetes, the KubeRay documentation provides an overview of how to configure the operator to execute and manage the Ray cluster lifecycle. This guide complements the KubeRay documentation by providing best practices for effectively using Ray deployments in your organization.
+
+This guide covers best practices for these deployment considerations:
+
+* Where to ship or run your code on the Ray cluster
+* Choosing a storage system for artifacts
+* Package dependencies for your application
+
+Deployment considerations are different for development and production. This table summarizes the recommended setup for both interactive development and production:
+
+|   | Interactive Development  | Production  |
+|---|---|---|
+| Cluster Configuration  | KubeRay YAML  | KubeRay YAML  |
+| Code | Run driver or Jupyter notebook on head node | S3 + runtime envs <br /> OR <br /> Bake code into Docker image (link)  |
```

Comment:
I removed the word, `link`.

Reply:
Building a Ray image from scratch is not easy, and our image-building CI pipelines are pretty complex. It will be helpful to have a doc in the future.

Reply:
https://docs.ray.io/en/master/serve/production-guide/docker.html => This is not enough. For example, some users are sensitive to security and want to build the image with different Linux distributions.